runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -1239,9 +1239,10 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
             TR::Node* parent = tt->getNode();
             TR::Node* callNode = tt->getNode()->getFirstChild();
             TR::SymbolReference* symRef =  callNode->getSymbolReference();
-            if (!callNode->getSymbolReference()->isUnresolved() && !visited.contains(callNode))
+            int i = callNode->getByteCodeIndex();
```

Comment:
I can see why you moved this out, but I worry that with such a common/simple name that is often used in loops we may get some weird interactions - could we just leave the `int i` inside and just add the extra call to `callNode->getByteCodeIndex()`. If we did want to pull it up we would need a better name IMO, but I don't think we want to consider a rename right now - just keep the scope as it was would be safest.

Reply:
changed as suggested.